htdocs/js/functions.js

```diff
@@ -193,11 +193,25 @@ vz.parseUrlParams = function() {
 
 				case 'from':
 				case 'to':
-					// disable automatic refresh
-					vz.options.refresh = false;
-					// ms or speaking timestamp
-					var ts = (/^-?[0-9]+$/.test(vars[key])) ? parseInt(vars[key]) : new Date(vars[key]).getTime();
-					if (!isNaN(ts)) {
+					if (key == 'to'){
+						// disable automatic refresh
+						vz.options.refresh = false;
+					}
+					// ms or speaking (relative) timestamp
+					var ts_param = vars[key];
+					var ts = null;
+					if (/^-?[0-9]+$/.test(ts_param)){ 
+						// string contains only numbers => it's a timestamp
+						ts = parseInt(ts_param);
+					} else {
+						// remove (outer) quotes from string which breaks strtotime (as expected)
```

Comment:
why is there need to remove quotes? Where do those come from?

Reply:
When parameters are put in quotes, like this: `http://vz.ip/?from="two days ago"` 

It works without this change if parameters are passed like this: `http://vz.ip/?from=two days ago` 

The latter is the recommended way for HTTP parameters as far as i understood it, but I also often experienced people putting the parameters in quotes, so I decided to catch that case. I might as well remove this, but then we surely have to document it somewhere explicitly because users will do it wrong for sure otherwise.

Reply:
Mhhm. Schadet sicher nix aber es gibt egtl. keinen Grund für die Quotes- nicht benötigt und gehören da nicht hin. Falls das häufig falsch gemacht wird schadet ein Workaround auch nix, ich würde es allerdings lieber "richtig" machen und das weg lassen. Wie Du willst.

Reply:
Würde ich dir zustimmen, ich werde es entfernen und dann kann man ja immer noch schauen. Werde den PR dieses WE fertigmachen.